plugin/mkdp.vim

```diff
@@ -43,24 +43,24 @@ if !exists('g:mkdp_command_for_global')
     let g:mkdp_command_for_global = 0
 endif
 
-function s:start_cmd(cmd, timer) abort
+function s:start_cmd(timer) abort
   if exists('*jobstart')
-    call jobstart(a:cmd)
+    call jobstart(s:start_cmd_value)
   elseif exists('*job_start')
-    call job_start(a:cmd)
+    call job_start(s:start_cmd_value)
   endif
 endfunction
 
 function! MKDP_browserfunc_default(url)
     " windows, including mingw
     if has('win32') || has('win64') || has('win32unix')
-        let l:cmd = 'cmd /c start ' . a:url . '.html'
+        let s:start_cmd_value = 'cmd /c start ' . a:url . '.html'
     " mac os
     elseif has('mac') || has('macunix') || has('gui_macvim') || system('uname') =~? '^darwin'
-        let l:cmd = 'open "' . a:url . '"'
+        let s:start_cmd_value = 'open "' . a:url . '"'
     " linux
     elseif executable('xdg-open')
-        let l:cmd = 'xdg-open "' . a:url . '"'
+        let s:start_cmd_value = 'xdg-open ' . a:url . ''
     " can not find the browser
```

Comment:
the `"` `"` is unnecessary for linux ?

Reply:
That's an interesting moment.
When I try to run with config which indicates browser 
`let g:mkdp_path_to_chrome`
All is working correctly because function `MKDP_browserfunc_default` not used.
But, when I trying to run without this value and with `"` `"`, I get this error:
```Unable to detect the URI-scheme of ""http://127.0.0.1:8708/markdown/15459000261?L2hvbWUvbXB5bHlwL0Rlc2t0b3AvdGVzdC5tZCY="".```
After I deleting `"` `"` all starts working...
When I just try run in terminal `xdg-open "[url address]"`, also all is working.
Looks like the program at some moment adds redundant `"` `"`.
For now, I don't know a better way to fix it, but I understand that is not the best way because it ruins a little bit uniformity of script.

Reply:
I found here is created redundant double quotes.
So, where is better to delete them?
[autoload/mkdp.vim#144](https://github.com/iamcco/markdown-preview.vim/blob/master/autoload/mkdp.vim#L144)

Reply:
I got the reason, it's because the difference of vim's `job_start` and neovim's `jobstart`, vim is not need for quotes.
so, it have to change like below

```viml
    " mac os
    elseif has('mac') || has('macunix') || has('gui_macvim') || system('uname') =~? '^darwin'
        if has('nvim')
              let s:start_cmd_value = 'open "' . a:url . '"'
           else
              let s:start_cmd_value = 'open ' . a:url . ''
        endif
    " linux
    elseif executable('xdg-open')
       if has('nvim')
            let s:start_cmd_value = 'xdg-open "' . a:url . '"'
         else
            let s:start_cmd_value = 'xdg-open ' . a:url . ''
       endif
```

> I found here is created redundant double quotes.
> So, where is better to delete them?
> [autoload/mkdp.vim#144](https://github.com/iamcco/markdown-preview.vim/blob/master/autoload/mkdp.vim#L144)

and this is not double quotes. because this is an execute command, so it have to have quotes for string params

Reply:
Yeah, with your code all started working correctly. I added them to pull request.
Oh, I got it, thank you.